docs/qc_metrics.md

```diff
@@ -0,0 +1,135 @@
+# PHGv2 - Quality Control Metrics
+
+In this document, we will discuss the built-in quality control 
+metrics available from the PHG. Metrics accompany several steps
+of building and using the PHG, and many can be run as either standalone 
+commands or as part of a related command.
+
+_WIP_
+
+## Anchowave Dotplots
+
+Anchorwave dotplots are a visual representation of the alignment between
+two assemblies. They are useful for identifying large-scale structural
+differences between assemblies, such as inversions, translocations, and
+large deletions.
+
+There are 2 methods to produce Anchorwave dotplots.
+
+1.  As part of 'align-assemblies':
+
+```shell
+./phg align-assemblies \
+    --gff data/anchors.gff \
+    --reference-file output/updated_assemblies/Ref.fa \
+    --assemblies data/assemblies_list.txt \
+    --total-threads 20 \
+    --in-parallel 2 \
+    -o output/alignment_files
+```
+
+The align-assemblies command runs code to create the anchorwave dotplots.  This is by default
+and is not optional.  The output is written to the specified output directory.  The dotplot files are
+stored as svg files with the name <assemblyName>_dotplot.svg.
+
+2. As a standalone command:
+
+```shell
+phg create-anchorwave-dotplot \
+    --input-file my.anchorspro \
+    --output-file myOutputFile.svg
+```
+
+The 'create-anshorwave-dotplot' command creates an anchorwave dotplot from the anchorwave created
+'anchorspro' file.  This file is generated by anchorwave when aligning assemblies.  Users may use the
+output from either the 'align-assemblies' command, or from any user run anchorwave alignment as the --input-file
+parameter to 'create-anchorwave-dotplot'.
+
+Output from 'create-anchorwave-dotplot' is written to the file specified by the --output-file parameter.  Users may
+specify output files with extension of .svg or .png and the appropriate file type will be generated.
+
+## VCF Metrics
+
+Once the gVCF and hVCF files for the TileDB instances have been created, 
+we can produce a table summarizing metrics related to the assemblies used to
+produce them. These metrics are useful for identifying low-quality assemblies 
+which you may wish to exclude from the PHG, or detecting problems with the
+assembly alignments.
+
+There are two methods to produce VCF metrics.
+
+1. As part of `create-maf-vcf`:
+
+```shell
+phg create-maf-vcf \
+    --db-path vcf_dbs \
+    --bed output/ref_ranges.bed \
+    --reference-file output/updated_assemblies/Ref.fa \
+    --maf-dir output/alignment_files \
+    -o output/vcf_files \
```

Comment:
It always seems confusing that most of the parameters are prefixed with 2 dashes, but output is with 1.  Perhaps that is just to accept.

Reply:
@lynnjo That is the convention for command line parameters. "long for of a flag" has 2 dashes. Short form (i.e. one letter) has 1 dash